themes/_core/js/checkout-payment.js

```diff
@@ -27,12 +27,12 @@ import prestashop from 'prestashop';
 
 class Payment {
   constructor() {
-    this.confirmationSelector = '#payment-confirmation';
-    this.conditionsSelector = '#conditions-to-approve';
-    this.conditionAlertSelector = '.js-alert-payment-conditions';
-    this.additionalInformatonSelector = '.js-additional-information';
-    this.optionsForm = '.js-payment-option-form';
-    this.termsCheckboxSelector = '#conditions-to-approve input[name="conditions_to_approve[terms-and-conditions]"]';
+    this.confirmationSelector = prestashop.selectors.checkout.confirmationSelector;
+    this.conditionsSelector = prestashop.selectors.checkout.conditionsSelector;
+    this.conditionAlertSelector = prestashop.selectors.checkout.conditionAlertSelector;
+    this.additionalInformatonSelector = prestashop.selectors.checkout.additionalInformatonSelector;
+    this.optionsForm = prestashop.selectors.checkout.optionsForm;
+    this.termsCheckboxSelector = prestashop.selectors.checkout.termsCheckboxSelector;
```

Comment:
Why not directly use `prestashop.selectors` rather assign it to an another value ?

Reply:
It could be a breaking chance